lua/lspconfig/pses.lua

```diff
@@ -0,0 +1,46 @@
+local configs = require 'lspconfig/configs'
+local util = require 'lspconfig/util'
+
+local server_name = "pses"
```

Comment:
Debating whether it should be powershell_es, because this will be the entry in table of contents so it might be more descriptive

Reply:
Agree